libbeat/cmd/instance/beat.go

```diff
@@ -608,6 +609,11 @@ func (b *Beat) configure(settings Settings) error {
 		return fmt.Errorf("error initializing logging: %v", err)
 	}
 
+	if b.Config.Migration.Enabled() {
+		// TODO: Not very nice way of doing this
+		template.Migration = true
```

Comment:
This is a pretty ugly hack to not have to pass this var through all stages up the the template alias processor. @urso I assume you don't like that?